CHANGELOG.md

```diff
@@ -14,7 +14,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 - Auditor+pyauditor: Added advanced filtering when querying records (#466) ([@raghuvar-vijay](https://github.com/raghuvar-vijay))
-- Auditor-pyauditor: Added bulk_insert option to insert list of records using auditor client and pyauditor (#580) ([@raghuvar-vijay](https://github.com/raghuvar-vijay))
+- Auditor+pyauditor: Added bulk_insert option to insert list of records using auditor client and pyauditor (#580) ([@raghuvar-vijay](https://github.com/raghuvar-vijay))
+- Auditor: Incorrect query string returns an empty vector (#598) ([@raghuvar-vijay](https://github.com/raghuvar-vijay))
```

Comment:
That's not 100% correct. Currently, I get back an error 500 with an error message, not an empty vector:

```
curl -v "localhost:8000/record?asdf=123"
...
< HTTP/1.1 500 Internal Server Error
< content-length: 49
< content-type: text/plain; charset=utf-8
< date: Wed, 10 Jan 2024 14:12:39 GMT
<
* Connection #0 to host localhost left intact
Query is incorrect, please check the query string
```

However, I think we should return an error `400 BAD REQUEST`. If we want to return a custom error message, I think we should to this via json, e.g.

```json
{
  "error": {
    "code": 400,
    "message": "Query is incorrect, please check the query string"
  }
}
```